src/BloomExe/BloomExe.csproj

```diff
@@ -802,14 +802,6 @@
       <SubType>Component</SubType>
     </Compile>
     <Compile Include="web\EnhancedImageServer.cs" />
-    <None Include="..\..\lib\dotnet\NDesk.DBus.dll" Condition="'$(OS)'=='Unix'">
-      <Link>NDesk.DBus.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\lib\dotnet\NDesk.DBus.dll.config" Condition="'$(OS)'=='Unix'">
-      <Link>NDesk.DBus.dll.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
```

Comment:
This probably isn't related to this commit. Ideally it would be its own commit. As it is, can you add a comment here so that we know removing ndesk was intentional?